phoenicis-repository/src/main/java/org/phoenicis/repository/RepositoryConfiguration.java

```diff
@@ -84,11 +81,22 @@ public RepositoryManager repositoryManager() {
                 backgroundRepositoryFactory());
 
         // set initial repositories
-        repositoryManager.addRepositories(this.loadRepositoryLocations().toArray(new RepositoryLocation[0]));
+        repositoryManager.addRepositories(this.repositoryLocationLoader().loadRepositoryLocations().toArray(new RepositoryLocation[0]));
 
         return repositoryManager;
     }
 
+    @Bean
+    public RepositoryLocationLoader repositoryLocationLoader() {
+        return new DynamicRepositoryLocationLoader(applicationContext, "filesystemLocationLoader");
+    }
+
+    @Bean
+    public RepositoryLocationLoader filesystemLocationLoader() {
+        return new FilesystemJsonRepositoryLocationLoader(repositoryListPath,
+                phoenicisGlobalConfiguration.objectMapper());
+    }
+
     public void saveRepositories(List<RepositoryLocation<? extends Repository>> repositoryLocations) {
```

Comment:
Can we add this method to `RepositoryLocationLoader`?

Reply:
Sure